Dockerfile

```diff
@@ -0,0 +1,41 @@
+# The MIT License
+#
+#  Copyright 2020 Loghi Perinpanayagam.
+#
+#  Permission is hereby granted, free of charge, to any person obtaining a copy
+#  of this software and associated documentation files (the "Software"), to deal
+#  in the Software without restriction, including without limitation the rights
+#  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+#  copies of the Software, and to permit persons to whom the Software is
+#  furnished to do so, subject to the following conditions:
+#
+#  The above copyright notice and this permission notice shall be included in
+#  all copies or substantial portions of the Software.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+#  THE SOFTWARE.
+
+
+FROM jenkins/ssh-agent
+
+# install python
+RUN apt-get update  && apt-get install -y \
+    python \
+    python-pip
+
+RUN apt install default-jdk -y
+
+# install dependencies
+RUN pip install ipython \
+    ipykernel \
+    jupyter-client \
+    protobuf \
+    grpcio
```

Comment:
We are installing `apt install python`, on Ubuntu, which is giving `python` version 2. `grpcio` appears to be Py>=3.5 only in PYPI.

![image](https://user-images.githubusercontent.com/304786/86862958-59eb6480-c11e-11ea-916a-4483688457be.png)

I've started a terminal within `jenkins/ssh-agent`, ran the `apt` commands, then tried installing with `pip install grpcio`

![image](https://user-images.githubusercontent.com/304786/86863087-9919b580-c11e-11ea-8d22-01dbc0be040a.png)

@Loghijiaha we will have to sort out what version of Python we need (I remember you mentioning somewhere zeppelin needed py 2?), update the version in the Dockerfile, and then continue looking at the permissions in the container.